SPECIFICATION.md

```diff
@@ -158,6 +159,73 @@ be on their own line.
 To prevent reference loops and to facilitate single-pass parsers, only backward
 references are allowed. All references glyphs must have been previously defined.
 
+#### Whitespaces
+
+Some glyphs need additional whitespace at the right of the glyph, or even
```

Comment:
Instead of "at the right", I'd write "following the glyph" or "after the glyph".